### PR TITLE
Fixed transformer

### PR DESCRIPTION
**Pull request by umbrella-kirill-starikov**
This PR
Fixed fatal error: call to a member function cloneNode() on null in ../InstantArticles/Transformer/Transformer.php on line 63.

### CODE REVIEW

**Comment by facebook-github-bot-5** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

**Comment by facebook-github-bot-6** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

File: src/Facebook/InstantArticles/Transformer/Getters/ElementGetter.php [hunk 1]
```diff
@@ -53,7 +53,7 @@ public function withSelector($selector)
     public function get($node)
     {
         $elements = self::findAll($node, $this->selector);
-        if (!empty($elements)) {
+        if (!empty($elements) and property_exists($elements, 'length') and $elements->length !== 0) {
```
Comment (everton-rosario) [inline src/Facebook/InstantArticles/Transformer/Getters/ElementGetter.php]: Can you please replace "and" with "&&"?
"and" isn't being used on the sdk code.

Thanks for your help!

**Comment by everton-rosario** [comment]
Nice catch @umbrella-kirill-starikov 
Just take a look on my inline comments regarding the new conditions.

Thanks for your help.

**Comment by umbrella-kirill-starikov** [comment]
@everton-rosario Hi, i'm replaced "and" with "&&". Thanks

**Pull request merged**